src/CalculatorUITestFramework/StandardOperatorsPanel.cs

```diff
@@ -0,0 +1,156 @@
+using OpenQA.Selenium.Appium.Windows;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CalculatorUITestFramework
+{
+    /// <summary>
+    /// UI elements and helper methods to perform common mathematical standard operations.
+    /// </summary>
+    public class StandardOperatorsPanel : ApplicationBase
+    {
+        public NumerPad NumberPad = new NumerPad();
+
+        public WindowsElement PercentButton => TryFindElementByAccessibilityId("percentButton");
+        public WindowsElement SquareRootButton => TryFindElementByAccessibilityId("squareRootButton");
+        public WindowsElement XPower2Button => TryFindElementByAccessibilityId("xpower2Button");
+        public WindowsElement XPower3Button => TryFindElementByAccessibilityId("xpower3Button");
+        public WindowsElement InvertButton => TryFindElementByAccessibilityId("invertButton");
+        public WindowsElement DivideButton => TryFindElementByAccessibilityId("divideButton");
+        public WindowsElement MultiplyButton => TryFindElementByAccessibilityId("multiplyButton");
+        public WindowsElement MinusButton => TryFindElementByAccessibilityId("minusButton");
+        public WindowsElement PlusButton => TryFindElementByAccessibilityId("plusButton");
+        public WindowsElement EqualButton => TryFindElementByAccessibilityId("equalButton");
+        public WindowsElement ClearEntryButton => TryFindElementByAccessibilityId("clearEntryButton");
+        public WindowsElement ClearButton => TryFindElementByAccessibilityId("clearButton");
+        public WindowsElement BackSpaceButton => TryFindElementByAccessibilityId("backSpaceButton");
+
+        /// <summary>
+        /// Uses the Calculator UI to add a list of numbers.
+        /// </summary>
+        /// <param name="numbers">List of numbers to be added.</param>
+        public void Add(List<double> numbers)
```

Comment:
Add, Subtract, etc. feel like the wrong level of abstraction to me. Some options I would like more:

1) remove a layer of abstraction and move this logic directly into the test code
2) create one method, with an implementation similar to NumberPad.TranslateToButtonClicks, which can handle all of these cases, like:
StandardOperatorsPanel.ClickButtons("1+2=");
StandardOperatorsPanel.ClickButtons("3-4=");